### PR TITLE
doc(standards): Add operator and golang standard as doing few improvements

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Keep the standards defined and centralized

## What
* add operator standards
* add go standards
* add adoc format for documentation as defined by the team docs
* add quay.io as standard for the images
* change the Readme format for adoc and add its standard with index and all

## Verification Steps
* Check [here](https://github.com/aerogear/standards/blob/4b402290fa8bab2b6a180161f58820628bd1aa61/README.adoc)

## Checklist:
- [X] Code has been tested locally by PR requester
- [ ]  Changes have been successfully verified by another team member

## Progress:
- [X] Finished task
- [ ] TODO

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+=== Cover Tests
+* https://coveralls.io/github/aerogear/[Coveralls.io]
+* https://codecov.io/gh/aerogear[Codecov.io]
+
+=== CI
+* https://shields.io/#/[Shields.io] - For customizations
+* https://circleci.com/[Circleci]
+* https://jenkins.io/[Jenkins]
+
+=== Versionalization
```
Comment (finp) [inline README.adoc]:
```suggestion
=== Versioning
```

Comment (finp) [inline README.adoc]:
```suggestion
=== Versioning
```

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+=== Java
+<- TODO ->
+
+=== Operators
+* https://github.com/operator-framework/operator-sdk[operator-sdk]
+* https://golang.org/[Golang] type/language
+* https://github.com/kubernetes/client-go[kubernetes/client-go]
+* https://kubernetes.io/docs/concepts/overview/kubernetes-api/[kubernetes API]
+
+== Scripts of this repository
```
Comment (finp) [inline README.adoc]:
```suggestion
== Scripts
```

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+* Execute following commands
+
+[source,shell]
+----
+npm i
+npm run getAll
+----
+
+* Check the file `allRepos.txt` created in the root directory of this project. This data can be used in the above command.
```
Comment (finp) [inline README.adoc]: how can it be used?

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+NOTE: This code implementation is using the https://github.com/octokit[octokit]
+
+TIP: See https://developer.github.com/v3[here] the GitHub API documentation
+
+NOTE: The repos will created ar `../build` directory
```
Comment (finp) [inline README.adoc]: I don't understand this note, typo?

File: README.adoc (hunk 3 above)
Comment (camilamacedo86) [reply README.adoc]: It is used to create PR with the github files for all repos from the org and/or defined.

File: README.adoc (hunk 1 above)
Comment (camilamacedo86) [reply README.adoc]: 👍

File: README.adoc (hunk 4 above)
Comment (camilamacedo86) [reply README.adoc]: 👍 Removed

**Review by davidffrench — APPROVED** [review]
Great work @camilamacedo86 . This looks good to me. It is very similar to the existing Readme with good additions.

**Pull request merged**